src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -1,37 +1,36 @@
+Use the following information to learn about backward incompatible changes that might require you to adjust Cloud configuration and processes for existing Cloud projects when you upgrade to the latest release of the {{site.data.var.ct}} package or other {{site.data.var.csuite}} packages.
 
 ### Service version requirement changes
 
-**PHP minimum version requirement**–In {{ site.data.var.ct }} v2002.1.0 and later, the minimum php version requirement is 7.1.3+. If your environment configuration specifies an earlier version, update the [php configuration]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options) in the `.magento.app.yaml` file. 
+We changed the minimum PHP version requirement from 7.1.0 to 7.1.3+ for Cloud projects that use `{{ site.data.var.ct }}` v2002.1.0 and later. If your environment configuration specifies an earlier version, update the [php configuration]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options) in the `.magento.app.yaml` file.
 
-### Environment variable changes
+### Environment configuration changes
 
-The following environment variables have been removed from the `{{site.var.data.ct}}` package:
-
--  `SCD_EXCLUDE_THEMES`–use the `SCD_MATRIX` variable instead.
--  `STATIC_CONTENT_THREADS`–use the `SCD_THREADS` variable the instead.
--  `DO_DEPLOY_STATIC_CONTENT`–use the `SKIP_SCD` variable instead.
--  `STATIC_CONTENT_SYMLINK`–Now, the `{{site.data.var.ct}}` always generates symlinks for static content in the `pub/static` directory when you generate static content during the build process.
+   Item | Status | Replacement
+   -------- |------- | -------
+   `SCD_EXCLUDE_THEMES` variable | Removed in `{{ site.data.var.ct }}` v2002.1.0 | [`SCD_MATRIX`]({{ site.baseurl}}/cloud/env/variables-build.html#scd_matrix)
+   `STATIC_CONTENT_THREADS` variable| Removed in `{{ site.data.var.ct }}` v2002.1.0 | [`SCD_THREADS`]({{ site.baseurl}}/cloud/env/variables-build.html#scd_threads)
+   `DO_DEPLOY_STATIC_CONTENT` variable | Removed in `{{ site.data.var.ct }}` v2002.1.0 | [`SKIP_SCD`]({{ site.baseurl}}/cloud/env/variables-build.html#skip_scd)
+   `STATIC_CONTENT_SYMLINK` variable | Removed in `{{site.data.var.ct}}` v2002.1.0 | None. Now, the build always creates a symlink to the static content directory `pub/static`.
+   `build_options.ini` file | Removed in `{{site.data.var.ct}}` v2002.1.0 | Use the [`.magento.env.yaml`]({{ site.baseurl }}/cloud/project/magento-env-yaml.html)) file to configure environment variables to manage build and deploy actions across all your environments.<br><br>If you build a Cloud environment that includes the `build_options.ini` file, the build fails.
 
 ### CLI command changes
```

Comment:
This needs to be clarified. These are used within deployment hooks (legacy command), not as stand-alone CLI commands. So the action item is - go to `.magento.app.yaml` and fix the hooks to new version. Please document what exactly is the new baseline: https://github.com/magento/magento-cloud/blob/master/.magento.app.yaml#L77-L88